gregorypoole.txt

```diff
@@ -0,0 +1,127 @@
+https://codepen.io/gregory-poole/pen/OJPwpdV?editors=1100
+
+
+1. What is Semantic HTML? 
+A way to organize html to make it make sense. 
+
+    2. What is HTML used for? 
+        html is used to create web pages
+
+    3. What is an attribute and where do we put it? 
+        a attribute can put a link inside of a tag.
+
+    4. What is the h1 tag used for? How many times should I use it on a page?
+    A h1 tag is used for a heading 
+
+    5. Name two tags that have required attributes
+    href and img src 
+
+    6. What do we put in the head of our HTML document? 
+    heading
+
```

Comment:
headings go in the body (which is confusing just based on the words 'head' and 'heading') Can you think of other things that may go in the head?